modules/nodes-containers-sysctls-about.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * nodes/containers/nodes-containers-sysctls.adoc
+
+[id="nodes-containers-sysctls-about_{context}"]
+= About sysctls
+
+In Linux, the sysctl interface allows an administrator to modify kernel
+parameters at runtime. Parameters are available via the *_/proc/sys/_* virtual
+process file system. The parameters cover various subsystems, such as:
+
+- kernel (common prefix: *_kernel._*)
+- networking (common prefix: *_net._*)
+- virtual memory (common prefix: *_vm._*)
+- MDADM (common prefix: *_dev._*)
+
+More subsystems are described in
+link:https://www.kernel.org/doc/Documentation/sysctl/README[Kernel documentation]. To get a list of all parameters, you can run:
+
+----
+$ sudo sysctl -a
+----
+
+[[undersatnding-sysctls]]
+== Understanding sysctls
+
+In Linux, the sysctl interface allows an administrator to modify kernel
+parameters at runtime. Parameters are available via the *_/proc/sys/_* virtual
+process file system. The parameters cover various subsystems, such as:
+
+- kernel (common prefix: *_kernel._*)
+- networking (common prefix: *_net._*)
+- virtual memory (common prefix: *_vm._*)
+- MDADM (common prefix: *_dev._*)
+
+More subsystems are described in
+link:https://www.kernel.org/doc/Documentation/sysctl/README[Kernel documentation]. To get a list of all parameters, you can run:
+
+----
+$ sudo sysctl -a
+----
+
+[[namespaced-vs-node-level-sysctls]]
+== Namespaced versus node-level sysctls
+
+A number of sysctls are _namespaced_ in today’s Linux kernels. This means that
+you can set them independently for each pod on a node. Being namespaced is a
+requirement for sysctls to be accessible in a pod context within Kubernetes.
+
+The following sysctls are known to be namespaced:
+
+- *_kernel.shm*_*
+- *_kernel.msg*_*
+- *_kernel.sem_*
+- *_fs.mqueue.*_*
+
+Additionally, most of the sysctls in the *_net.*_* group are known
+to be namespaced. Their namespace adoption differs based on the kernel
+version and distributor.
+
+To check which *_net.*_* sysctls are namespaced on your system, run the
+following command:
+
+----
+$ podman run --rm -ti docker.io/fedora \
```

Comment:
this section on running podman is awesome, but is somewhat contrary to us not wanting customers to SSH into machines. I vote on removing this blurb on checking the namespaced sysctls.